returnn/tf/layers/signal_processing.py

```diff
@@ -31,21 +30,29 @@ def __init__(self, **kwargs):
 
   @classmethod
   def get_out_data_from_opts(cls, name, sources, n_out=None, **kwargs):
-    return super(AlternatingRealToComplexLayer, cls).get_out_data_from_opts(name=name, sources=sources, out_type={"dim": n_out, "dtype": "complex64", "batch_dim_axis": 0, "time_dim_axis": 1}, **kwargs)
+    """
+    :param str name:
+    :param list[LayerBase] sources:
+    :param int|None|NotSpecified n_out:
```

Comment:
`NotSpecified` is not imported here, so you cannot refer to it directly like that. (You will see that PyCharm cannot correctly infer the type, when you move the mouse over it, and try to ctrl+click on it .)
```suggestion
    :param int|None|returnn.util.basic.NotSpecified n_out:
```